components/TARGET_PSA/inc/psa/protected_storage.h

```diff
@@ -78,8 +78,8 @@ psa_status_t psa_ps_set(psa_storage_uid_t uid,
  * \retval      PSA_ERROR_INVALID_SIGNATURE  The operation failed because the data associated with the UID failed authentication
  */
 psa_status_t psa_ps_get(psa_storage_uid_t uid,
-                        uint32_t data_offset,
-                        uint32_t data_length,
+                        size_t data_offset,
+                        size_t data_length,
                         void *p_data);
```

Comment:
According to the spec version you pointed at, `psa_ps_get()` needs updating to match the following function signature:
```C
psa_status_t psa_ps_get(psa_storage_uid_t uid,
                        size_t data_offset,
                        size_t data_size,
                        void *p_data,
                        size_t *p_data_length );
```

It looks like `p_data_length` is currently missing.

Reply:
Will change.